timers.js

```diff
@@ -1,4 +1,76 @@
+// Step 1: Create JS selectors to target each of the timer buttons
+var reset = document.querySelector("#reset");
+var start = document.querySelector("#start");
+var pause = document.querySelector("#pause");
+var timer = document.querySelector("#timer");
 
+//  Step 3: Instantiate seconds and timerId veriables
+var seconds = 0;
+var timerId = 0;
 
 
+// Step 2. create click handlers for the timer buttons
+//reset.addEventListener("click",);
 
+
+
+
+// Step 4: Create function updateTime
+  // increment by seconds (1)
+  // insert value of seconds into <h1> element with id="#timer"
+
+
+function updateTime () {
+  seconds++;
+}
+
+// Start Functionality
+  // click start
+    //  replace text "Stop Watch" with "Time elapsed: 0"
+    // increase count by 1 second.
+
+    function startClick () {
+      var newSeconds = seconds++;
+      timer.innerHTML = "Time elapsed: " + newSeconds;
+      timerId = setInterval(updateTime, 1000)
+      console.log("start action happens")
```

Comment:
when we use a `setInterval` the return value is an integer value representative of that interval's id. This is how we can point to this interval later